login-workflow/src/screens/SiteOptionsScreen/SiteOptionsScreenBase.tsx

```diff
@@ -0,0 +1,127 @@
+ * @param message success message to be displayed on the screen
+ * @param joinExistingOrgLabel to display label for the custom button which redirects to existing organization flow
+ * @param createNewOrgLabel display label for the custom button which redirects to create a new organization flow
+ * @param onJoinExistingOrg function to call when user wants to join existing organization
+ * @param onCreateNewOrg function to call when user wants to create a new  organization
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ *
+ * @category Component
+ */
+
+export const SiteOptionsScreenBase: React.FC<React.PropsWithChildren<SiteOptionsScreenProps>> = (props) => {
+    const {
+        icon,
+        message = '',
+        joinExistingOrgLabel = '',
+        createNewOrgLabel = '',
+        canCreateNewOrg,
+        canJoinExistingOrg,
+        onJoinExistingOrg,
+        onCreateNewOrg,
+    } = props;
+    const cardBaseProps = props.WorkflowCardBaseProps || {};
+    const headerProps = props.WorkflowCardHeaderProps || {};
+    const instructionsProps = props.WorkflowCardInstructionProps || {};
+    return (
+        <WorkflowCard {...cardBaseProps}>
+            <WorkflowCardHeader {...headerProps} />
+            {Object.keys(instructionsProps).length !== 0 && <WorkflowCardInstructions {...instructionsProps} />}
+            <WorkflowCardBody>
+                <Box
+                    sx={{
+                        display: 'flex',
+                        flexDirection: 'column',
+                        alignItems: 'center',
+                        justifyContent: 'center',
+                        height: '100%',
+                    }}
+                >
+                    {icon && (
+                        <Box
+                            sx={{
+                                display: 'flex',
+                                justifyContent: 'center',
+                                mb: 6,
+                            }}
+                        >
+                            {icon}
+                        </Box>
+                    )}
+                    {message && (
+                        <Box
+                            sx={{
+                                flexDirection: 'column',
+                                alignItems: 'center',
+                                justifyContent: 'center',
+                            }}
+                        >
+                            {message}
+                        </Box>
+                    )}
+                </Box>
+            </WorkflowCardBody>
+            <Box
+                sx={{
+                    p: { xs: 2, md: 3 },
+                }}
+            >
+                <Divider />
```

Comment:
The screenshot in miro is wrong and this divider needs to span the entire width of the page